sdk/include/psdk/ws2tcpip.h

```diff
@@ -451,15 +451,15 @@ PCSTR
 WSAAPI
 inet_ntop(
   _In_ INT Family,
-  _In_ PVOID pAddr,
+  _In_ const PVOID pAddr,
```

Comment:
Actually, for this function and the next one, this should be `const VOID*`, because it should be a _pointer to **`const VOID`**_ i.e. the pointer is what it is, but it points to some memory area that needs to be considered as const (won't be changed by the function).
Alternatively one could use `PCVOID` there.

Official headers use `const VOID*` here (and the other function), so please use that instead.
This change should also be applied to the prototypes of these two functions in dll/win32/ws2_32/src/addrconv.c